solidity/test/XConversions.js

```diff
@@ -137,27 +131,10 @@ contract("XConversions", accounts => {
 
             const prevBalance = await erc20Token.balanceOf(accounts[5])
 
-            const retAmount = await bntConverter.completeXConversion.call(
-                path,                                              
-                1,                                               
-                xTransferId,                                       
-                0,                                                    
-                0,                                                      
-                utils.zeroBytes32,                                                      
-                utils.zeroBytes32,                                                      
-                { from: accounts[5] }
-            )
-
-            const res = await bntConverter.completeXConversion(
-                path,                                              
-                1,                                               
-                xTransferId,                                       
-                0,                                                    
-                0,                                                      
-                utils.zeroBytes32,                                                      
-                utils.zeroBytes32,                                                      
-                { from: accounts[5] }
-            )
+            await bntToken.approve(bancorNetwork.address, 0, { from: accounts[5] })
```

Comment:
@yudilevi : This line implies a stateful test. How come the allowance is larger than 0 (to require executing this "0-approve" transaction)? We should strive for stateless tests ("it"s).